qa/rolling-upgrade/src/test/resources/processor/IndexMappings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "knn.algo_param.ef_search": 100,
+      "refresh_interval": "30s",
+      "default_pipeline": "%s"
+    },
+    "number_of_shards": 6,
+    "number_of_replicas": 0
```

Comment:
same comment as of in RestartUpgrade

Reply:
https://github.com/opensearch-project/neural-search/pull/515#discussion_r1437284537